pallets/pallet-did-lookup/src/benchmarking.rs

```diff
@@ -57,70 +65,214 @@ benchmarks! {
 		let previous_did: T::DidIdentifier = account("prev", 0, SEED + 1);
 		let connected_acc = sr25519_generate(KeyTypeId(*b"aura"), None);
 		let connected_acc_id: T::AccountId = connected_acc.into();
+		let linkable_id: LinkableAccountId = connected_acc_id.clone().into();
```

Comment:
I guess we don't need the `associate_account` benchmark case anymore since we have all the other cases covered below.